src/server/players/services/scoreboard.ts

```diff
@@ -8,16 +9,30 @@ export async function initScoreboardService() {
 		stats.Name = "leaderstats";
 		stats.Parent = player;
 
+		const knockouts = new Instance("IntValue");
+		knockouts.Name = "☠️ KOs";
+		knockouts.Parent = stats;
+
 		const score = new Instance("IntValue");
-		score.Name = "Score";
+		score.Name = "💯 Score";
 		score.Parent = stats;
 
-		const unsubscribe = store.subscribe(selectSnakeById(player.Name), (snake) => {
+		const cash = new Instance("IntValue");
+		cash.Name = "💵 Cash";
+		cash.Parent = stats;
+
+		const unsubscribeFromScoreAndKnockoutsUpdate = store.subscribe(selectSnakeById(player.Name), (snake) => {
```

Comment:
These variable names are rather verbose -- consider `unsubscribeFromSnake` and `unsubscribeFromCash` instead